change/just-scripts-2020-07-14-16-35-05-vipati-eslint-timing-fix.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Setting Timing value to environment variable.",
```

It's good to include the specific task you're updating in the change description.
```suggestion
  "comment": "ESLint task: Setting Timing value to environment variable.",
```